nixos/extractors/services.nix

```diff
@@ -36,6 +39,29 @@ in {
           details.listen = mkIf (address != null && port != null) {text = "${address}:${toString port}";};
         };
 
+      authelia = let
+        instances =
+          filterAttrs (_: v: v.enable) config.services.authelia.instances;
+      in
+        mkIf (instances != {}) {
+          name = "Authelia";
+          icon = "services.authelia";
+          details.listen.text =
```

Comment:
Is it a common thing to use multiple instances in authelia? If so, putting all that listen information in a single detail value will probably cause a lot of word wrapping and make it hard to read.

In theory you could set one detail for each instance `details."${instanceName}.listen" = ${instanceListenValue}`, that will allocate at least one line for each one. But it's up to you whether you want to do that.

Reply:
Thanks! Should be resolved via https://github.com/oddlama/nix-topology/pull/4/commits/d44c6cf8919d77d405a873ae10376680b816b40e